service/cloudconfigv4/master_template.go

```diff
@@ -243,8 +246,11 @@ func (e *MasterExtension) Units() ([]k8scloudconfig.UnitAsset, error) {
 		{
 			AssetContent: decryptKeysServiceTemplate,
 			Name:         "decrypt-keys-assets.service",
-			Enable:       true,
-			Command:      "start",
+			// Do not enable key decrypt unit so that it won't get autmatically
+			// executed on master reboot. This will prevent eventual races with the
+			// key files creation.
+			Enable:  false,
```

Comment:
So when it is disabled by default who or what starts it and when?

Reply:
It is started by `cloud-init`, we don't enable it but keep the command as `start`, so that it is run on this boot but won't be set to autostart on next boot. The decrypt units are extensions added in aws-operator, k8scc creates the actual units here: https://github.com/giantswarm/k8scloudconfig/blob/master/v_3_0_0/master_template.go#L1835 you can see there that the `command` field comes from the extension.

There's something to note here, we run cloud-init on each boot, so, regardless of the decrypt units being enabled or disabled, they will be run on each boot anyway. The difference when they are disabled is that they run always after being started by cloud-init, and the race is prevented.

Reply:
Thanks for this work. In general, approach is fine. But cloud-init is "ugly" (your note above about reboot case).

In my opinion, proper solution is to make unit itself idempotent.

You can add
```
[Unit]
ConditionPathExists=/etc/kubernetes/ssl/somekey.pem
```

or modify script to check if there is anything to decrypt there.

Reply:
@r7vme thx a lot for your comments. IMO your suggestion wouldn't fix the race, the condition you propose will always be met, but it is possible that the certificates to decrypt are not the correct ones. On first boot there are no certificates and the decrypts units are not created yet. Then, `oem-cloudinit.service` kicks in, this is its log:
```
$ sudo journalctl -u oem-cloudinit.service
-- Logs begin at Fri 2018-01-26 16:57:07 UTC, end at Fri 2018-01-26 17:30:31 UTC. --
Jan 26 16:57:16 localhost systemd[1]: Starting Cloudinit from EC2 metadata...
Jan 26 16:57:16 localhost coreos-cloudinit[772]: 2018/01/26 16:57:16 Checking availability of "cloud-drive"
Jan 26 16:57:16 localhost coreos-cloudinit[772]: 2018/01/26 16:57:16 Checking availability of "ec2-metadata-service"
Jan 26 16:57:16 localhost coreos-cloudinit[772]: 2018/01/26 16:57:16 Fetching user-data from datasource of type "ec2-metadata-service"
Jan 26 16:57:16 localhost coreos-cloudinit[772]: 2018/01/26 16:57:16 Fetching data from http://169.254.169.254/2009-04-04/user-data. Attempt #1
Jan 26 16:57:16 localhost coreos-cloudinit[772]: 2018/01/26 16:57:16 Fetching meta-data from datasource of type "ec2-metadata-service"
Jan 26 16:57:16 localhost coreos-cloudinit[772]: 2018/01/26 16:57:16 Fetching data from http://169.254.169.254/2009-04-04/meta-data/public-keys. Attempt #1
Jan 26 16:57:16 localhost coreos-cloudinit[772]: 2018/01/26 16:57:16 Fetching data from http://169.254.169.254/2009-04-04/meta-data/hostname. Attempt #1
Jan 26 16:57:16 localhost coreos-cloudinit[772]: 2018/01/26 16:57:16 Fetching data from http://169.254.169.254/2009-04-04/meta-data/local-ipv4. Attempt #1
Jan 26 16:57:16 localhost coreos-cloudinit[772]: 2018/01/26 16:57:16 Fetching data from http://169.254.169.254/2009-04-04/meta-data/public-ipv4. Attempt #1
Jan 26 16:57:16 localhost coreos-cloudinit[772]: 2018/01/26 16:57:16 Parsing user-data as script
Jan 26 16:57:16 localhost coreos-cloudinit[772]: 2018/01/26 16:57:16 Merging cloud-config from meta-data and user-data
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal coreos-cloudinit[772]: 2018/01/26 16:57:17 Set hostname to ip-10-12-1-88.eu-central-1.compute.internal
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal coreos-cloudinit[772]: 2018/01/26 16:57:17 Writing file to "/etc/environment"
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal coreos-cloudinit[772]: 2018/01/26 16:57:17 Wrote file to "/etc/environment"
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal coreos-cloudinit[772]: 2018/01/26 16:57:17 Updated /etc/environment
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal coreos-cloudinit[772]: 2018/01/26 16:57:17 Ensuring runtime unit file "etcd.service" is unmasked
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal coreos-cloudinit[772]: 2018/01/26 16:57:17 Ensuring runtime unit file "etcd2.service" is unmasked
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal coreos-cloudinit[772]: 2018/01/26 16:57:17 Ensuring runtime unit file "fleet.service" is unmasked
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal coreos-cloudinit[772]: 2018/01/26 16:57:17 Ensuring runtime unit file "locksmithd.service" is unmasked
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal coreos-cloudinit[772]: 2018/01/26 16:57:17 Writing file to "/var/lib/coreos-cloudinit/scripts/476003234"
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal coreos-cloudinit[772]: 2018/01/26 16:57:17 Wrote file to "/var/lib/coreos-cloudinit/scripts/476003234"
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal coreos-cloudinit[772]: 2018/01/26 16:57:17 Creating transient systemd unit 'coreos-cloudinit-476003234.service'
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal coreos-cloudinit[772]: 2018/01/26 16:57:17 Writing file to "/var/lib/coreos-cloudinit/scripts/unit-name"
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal coreos-cloudinit[772]: 2018/01/26 16:57:17 Wrote file to "/var/lib/coreos-cloudinit/scripts/unit-name"
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal systemd[1]: Started Cloudinit from EC2 metadata.
```
You can see that it pulls the instance's user data and then creates a transient `coreos-cloudinit-476003234.service` unit, which in turn executes our `master.yaml`, this is its log:
```
-- Logs begin at Fri 2018-01-26 16:57:07 UTC, end at Fri 2018-01-26 17:32:38 UTC. --
Jan 26 16:57:17 ip-10-12-1-88.eu-central-1.compute.internal systemd[1]: Started Unit generated and executed by coreos-cloudinit on behalf of user.
Jan 26 16:57:24 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: pubkey: prefix: "quay.io/coreos/awscli"
Jan 26 16:57:24 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: key: "https://quay.io/aci-signing-key"
Jan 26 16:57:24 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: gpg key fingerprint is: BFF3 13CD AA56 0B16 A898  7B8F 72AB F5F6 799D 33BC
Jan 26 16:57:24 ip-10-12-1-88.eu-central-1.compute.internal bash[808]:         Quay.io ACI Converter (ACI conversion signing key) <support@quay.io>
Jan 26 16:57:24 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: Trusting "https://quay.io/aci-signing-key" for prefix "quay.io/coreos/awscli" without fingerprint review.
Jan 26 16:57:24 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: Added key for prefix "quay.io/coreos/awscli" at "/etc/rkt/trustedkeys/prefix.d/quay.io/coreos/awscli/bff313cdaa560b16a8987b8f72abf5f6799d33bc"
Jan 26 16:57:24 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: Downloading signature:  0 B/473 B
Jan 26 16:57:24 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: Downloading signature:  473 B/473 B
Jan 26 16:57:24 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: Downloading signature:  473 B/473 B
Jan 26 16:57:26 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: Downloading ACI:  0 B/32 MB
Jan 26 16:57:26 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: Downloading ACI:  8.19 KB/32 MB
Jan 26 16:57:27 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: Downloading ACI:  15.8 MB/32 MB
Jan 26 16:57:27 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: Downloading ACI:  32 MB/32 MB
Jan 26 16:57:28 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: image: signature verified:
Jan 26 16:57:28 ip-10-12-1-88.eu-central-1.compute.internal bash[808]:   Quay.io ACI Converter (ACI conversion signing key) <support@quay.io>
Jan 26 16:57:35 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: [174B blob data]
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Checking availability of "local-file"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Fetching user-data from datasource of type "local-file"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 line 2582: warning: unrecognized key "storage"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Fetching meta-data from datasource of type "local-file"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Parsing user-data as cloud-config
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Merging cloud-config from meta-data and user-data
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Creating user 'test-user'
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal useradd[928]: new group: name=test-user, GID=1000
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal useradd[928]: new user: name=test-user, UID=1000, GID=1000, home=/home/test-user, shell=/bin/bash
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal useradd[928]: add 'test-user' to group 'sudo'
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal useradd[928]: add 'test-user' to group 'docker'
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal useradd[928]: add 'test-user' to shadow group 'sudo'
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal useradd[928]: add 'test-user' to shadow group 'docker'
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Authorizing 1 SSH keys for user 'test-user'
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/calico-kube-controllers-sa.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/calico-kube-controllers-sa.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/calico-kube-controllers-sa.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/calico-node-sa.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/calico-node-sa.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/calico-node-sa.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/calico-configmap.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/calico-configmap.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/calico-configmap.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/calico-ds.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/calico-ds.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/calico-ds.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/calico-kube-controllers.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/calico-kube-controllers.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/calico-kube-controllers.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/coredns.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/coredns.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/coredns.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/network-policy.json"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/network-policy.json"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/network-policy.json to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/default-backend-dep.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/default-backend-dep.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/default-backend-dep.yml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/default-backend-svc.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/default-backend-svc.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/default-backend-svc.yml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/ingress-controller-cm.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/ingress-controller-cm.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/ingress-controller-cm.yml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/ingress-controller-dep.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/ingress-controller-dep.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/ingress-controller-dep.yml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/ingress-controller-svc.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/ingress-controller-svc.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/ingress-controller-svc.yml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/kube-proxy-sa.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/kube-proxy-sa.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/kube-proxy-sa.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/kube-proxy-ds.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/kube-proxy-ds.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/kube-proxy-ds.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/node-exporter-svc.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/node-exporter-svc.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/node-exporter-svc.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/node-exporter-sa.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/node-exporter-sa.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/node-exporter-sa.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/node-exporter-ds.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/node-exporter-ds.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/node-exporter-ds.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/kube-state-metrics-svc.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/kube-state-metrics-svc.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/kube-state-metrics-svc.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/kube-state-metrics-sa.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/kube-state-metrics-sa.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/kube-state-metrics-sa.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/kube-state-metrics-dep.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/kube-state-metrics-dep.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/kube-state-metrics-dep.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/rbac_bindings.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/rbac_bindings.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/rbac_bindings.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/rbac_roles.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/rbac_roles.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/rbac_roles.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/psp_policies.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/psp_policies.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/psp_policies.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/psp_roles.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/psp_roles.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/psp_roles.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/psp_binding.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/psp_binding.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/psp_binding.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/opt/wait-for-domains"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/opt/wait-for-domains"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /opt/wait-for-domains to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/opt/k8s-addons"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/opt/k8s-addons"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /opt/k8s-addons to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/config/addons-kubeconfig.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/config/addons-kubeconfig.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/config/addons-kubeconfig.yml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/config/proxy-kubeconfig.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/config/proxy-kubeconfig.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/config/proxy-kubeconfig.yml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/config/kubelet-kubeconfig.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/config/kubelet-kubeconfig.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/config/kubelet-kubeconfig.yml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/config/controller-manager-kubeconfig.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/config/controller-manager-kubeconfig.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/config/controller-manager-kubeconfig.yml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/config/scheduler-kubeconfig.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/config/scheduler-kubeconfig.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/config/scheduler-kubeconfig.yml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/encryption/k8s-encryption-config.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/encryption/k8s-encryption-config.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/encryption/k8s-encryption-config.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/manifests/audit-policy.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/manifests/audit-policy.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/manifests/audit-policy.yml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/ssh/sshd_config"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/ssh/sshd_config"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/ssh/sshd_config to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/sysctl.d/hardening.conf"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/sysctl.d/hardening.conf"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/sysctl.d/hardening.conf to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/audit/rules.d/10-docker.rules"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/audit/rules.d/10-docker.rules"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/audit/rules.d/10-docker.rules to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/audit-rules.service.d/10-Wait-For-Docker.conf"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/audit-rules.service.d/10-Wait-For-Docker.conf"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/systemd/system/audit-rules.service.d/10-Wait-For-Docker.conf to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/opt/bin/decrypt-tls-assets"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/opt/bin/decrypt-tls-assets"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /opt/bin/decrypt-tls-assets to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/apiserver-crt.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/apiserver-crt.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/apiserver-crt.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/apiserver-ca.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/apiserver-ca.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/apiserver-ca.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/apiserver-key.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/apiserver-key.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/apiserver-key.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/service-account-crt.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/service-account-crt.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/service-account-crt.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/service-account-ca.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/service-account-ca.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/service-account-ca.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/service-account-key.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/service-account-key.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/service-account-key.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/calico/client-crt.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/calico/client-crt.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/calico/client-crt.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/calico/client-ca.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/calico/client-ca.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/calico/client-ca.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/calico/client-key.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/calico/client-key.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/calico/client-key.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/etcd/server-crt.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/etcd/server-crt.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/etcd/server-crt.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/etcd/server-ca.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/etcd/server-ca.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/etcd/server-ca.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/etcd/server-key.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/etcd/server-key.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/etcd/server-key.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/etcd/client-crt.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/etcd/client-crt.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/etcd/client-crt.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/etcd/client-ca.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/etcd/client-ca.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/etcd/client-ca.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/ssl/etcd/client-key.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/ssl/etcd/client-key.pem.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/ssl/etcd/client-key.pem.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/docker.service.d/01-wait-docker.conf"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/docker.service.d/01-wait-docker.conf"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/systemd/system/docker.service.d/01-wait-docker.conf to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/opt/bin/decrypt-keys-assets"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/opt/bin/decrypt-keys-assets"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /opt/bin/decrypt-keys-assets to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/kubernetes/encryption/k8s-encryption-config.yaml.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/kubernetes/encryption/k8s-encryption-config.yaml.enc"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/kubernetes/encryption/k8s-encryption-config.yaml.enc to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/ingress-controller-cm.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/ingress-controller-cm.yml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/ingress-controller-cm.yml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/srv/default-storage-class.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/srv/default-storage-class.yaml"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /srv/default-storage-class.yaml to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/coreos/update.conf"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/coreos/update.conf"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file /etc/coreos/update.conf to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "decrypt-tls-assets.service" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/decrypt-tls-assets.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/decrypt-tls-assets.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "decrypt-tls-assets.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "decrypt-tls-assets.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "decrypt-tls-assets.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "format-var-lib-docker.service" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/format-var-lib-docker.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/format-var-lib-docker.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "format-var-lib-docker.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "format-var-lib-docker.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "format-var-lib-docker.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "var-lib-docker.mount" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/var-lib-docker.mount"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/var-lib-docker.mount"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "var-lib-docker.mount"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "var-lib-docker.mount"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "var-lib-docker.mount"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "decrypt-keys-assets.service" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/decrypt-keys-assets.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/decrypt-keys-assets.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "decrypt-keys-assets.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "decrypt-keys-assets.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "decrypt-keys-assets.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "format-etcd-ebs.service" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/format-etcd-ebs.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/format-etcd-ebs.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "format-etcd-ebs.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "format-etcd-ebs.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "format-etcd-ebs.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "etc-kubernetes-data-etcd.mount" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/etc-kubernetes-data-etcd.mount"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/etc-kubernetes-data-etcd.mount"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "etc-kubernetes-data-etcd.mount"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "etc-kubernetes-data-etcd.mount"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "etc-kubernetes-data-etcd.mount"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "wait-for-domains.service" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/wait-for-domains.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/wait-for-domains.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "wait-for-domains.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "wait-for-domains.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "wait-for-domains.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "os-hardeing.service" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/os-hardeing.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/os-hardeing.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "os-hardeing.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "os-hardeing.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "os-hardeing.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "set-ownership-etcd-data-dir.service" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/set-ownership-etcd-data-dir.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/set-ownership-etcd-data-dir.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "set-ownership-etcd-data-dir.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "set-ownership-etcd-data-dir.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "set-ownership-etcd-data-dir.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing drop-in unit "10-giantswarm-extra-args.conf" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/docker.service.d/10-giantswarm-extra-args.conf"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/docker.service.d/10-giantswarm-extra-args.conf"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote drop-in unit "10-giantswarm-extra-args.conf"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "docker.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "docker.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "k8s-setup-network-env.service" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/k8s-setup-network-env.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/k8s-setup-network-env.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "k8s-setup-network-env.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "k8s-setup-network-env.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "k8s-setup-network-env.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Masking unit file "etcd2.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "etcd3.service" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/etcd3.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/etcd3.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "etcd3.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "etcd3.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "etcd3.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "etcd3-defragmentation.service" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/etcd3-defragmentation.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/etcd3-defragmentation.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "etcd3-defragmentation.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "etcd3-defragmentation.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "etcd3-defragmentation.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "etcd3-defragmentation.timer" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/etcd3-defragmentation.timer"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/etcd3-defragmentation.timer"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "etcd3-defragmentation.timer"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "etcd3-defragmentation.timer"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "etcd3-defragmentation.timer"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "k8s-kubelet.service" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/k8s-kubelet.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/k8s-kubelet.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "k8s-kubelet.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "k8s-kubelet.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "k8s-kubelet.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Masking unit file "update-engine.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Masking unit file "locksmithd.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Masking unit file "fleet.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Masking unit file "fleet.socket"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Masking unit file "flanneld.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "systemd-networkd-wait-online.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabled unit "systemd-networkd-wait-online.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing unit "k8s-api-server.service" to filesystem
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Writing file to "/etc/systemd/system/k8s-api-server.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote file to "/etc/systemd/system/k8s-api-server.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Wrote unit "k8s-api-server.service"
Jan 26 16:57:36 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:36 Enabling unit file "k8s-api-server.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Enabled unit "k8s-api-server.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Writing unit "k8s-controller-manager.service" to filesystem
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Writing file to "/etc/systemd/system/k8s-controller-manager.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Wrote file to "/etc/systemd/system/k8s-controller-manager.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Wrote unit "k8s-controller-manager.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Enabling unit file "k8s-controller-manager.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Enabled unit "k8s-controller-manager.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Writing unit "k8s-scheduler.service" to filesystem
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Writing file to "/etc/systemd/system/k8s-scheduler.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Wrote file to "/etc/systemd/system/k8s-scheduler.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Wrote unit "k8s-scheduler.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Enabling unit file "k8s-scheduler.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Enabled unit "k8s-scheduler.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Writing unit "k8s-addons.service" to filesystem
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Writing file to "/etc/systemd/system/k8s-addons.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Wrote file to "/etc/systemd/system/k8s-addons.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Wrote unit "k8s-addons.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Enabling unit file "k8s-addons.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Enabled unit "k8s-addons.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Ensuring runtime unit file "etcd.service" is unmasked
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Ensuring runtime unit file "etcd2.service" is unmasked
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Ensuring runtime unit file "fleet.service" is unmasked
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Ensuring runtime unit file "locksmithd.service" is unmasked
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Masking unit file "locksmithd.service"
Jan 26 16:57:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:37 Calling unit command "start" on "decrypt-tls-assets.service"
Jan 26 16:57:43 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:43 Result of "start" on "decrypt-tls-assets.service": done
Jan 26 16:57:43 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:43 Calling unit command "start" on "format-var-lib-docker.service"
Jan 26 16:57:43 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:43 Result of "start" on "format-var-lib-docker.service": done
Jan 26 16:57:43 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:43 Calling unit command "start" on "var-lib-docker.mount"
Jan 26 16:57:43 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:43 Result of "start" on "var-lib-docker.mount": done
Jan 26 16:57:43 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:43 Calling unit command "start" on "decrypt-keys-assets.service"
Jan 26 16:57:44 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:44 Result of "start" on "decrypt-keys-assets.service": done
Jan 26 16:57:44 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:44 Calling unit command "start" on "format-etcd-ebs.service"
Jan 26 16:57:47 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:47 Result of "start" on "format-etcd-ebs.service": done
Jan 26 16:57:47 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:47 Calling unit command "start" on "etc-kubernetes-data-etcd.mount"
Jan 26 16:57:48 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:48 Result of "start" on "etc-kubernetes-data-etcd.mount": done
Jan 26 16:57:48 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:48 Calling unit command "start" on "wait-for-domains.service"
Jan 26 16:57:48 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:48 Result of "start" on "wait-for-domains.service": done
Jan 26 16:57:48 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:48 Calling unit command "start" on "os-hardeing.service"
Jan 26 16:57:48 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:48 Result of "start" on "os-hardeing.service": done
Jan 26 16:57:48 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:48 Calling unit command "start" on "set-ownership-etcd-data-dir.service"
Jan 26 16:57:48 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:48 Result of "start" on "set-ownership-etcd-data-dir.service": done
Jan 26 16:57:48 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:48 Calling unit command "start" on "docker.service"
Jan 26 16:57:49 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:49 Result of "start" on "docker.service": done
Jan 26 16:57:49 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:49 Calling unit command "start" on "k8s-setup-network-env.service"
Jan 26 16:57:54 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:54 Result of "start" on "k8s-setup-network-env.service": done
Jan 26 16:57:54 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:54 Calling unit command "stop" on "etcd2.service"
Jan 26 16:57:54 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:54 Result of "stop" on "etcd2.service": done
Jan 26 16:57:54 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:57:54 Calling unit command "start" on "etcd3.service"
Jan 26 16:58:00 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:00 Result of "start" on "etcd3.service": done
Jan 26 16:58:00 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:00 Calling unit command "start" on "etcd3-defragmentation.service"
Jan 26 16:58:01 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:01 Result of "start" on "etcd3-defragmentation.service": done
Jan 26 16:58:01 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:01 Calling unit command "start" on "etcd3-defragmentation.timer"
Jan 26 16:58:01 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:01 Result of "start" on "etcd3-defragmentation.timer": done
Jan 26 16:58:01 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:01 Calling unit command "start" on "k8s-kubelet.service"
Jan 26 16:58:32 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:32 Result of "start" on "k8s-kubelet.service": done
Jan 26 16:58:32 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:32 Calling unit command "stop" on "update-engine.service"
Jan 26 16:58:33 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:33 Result of "stop" on "update-engine.service": done
Jan 26 16:58:33 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:33 Calling unit command "stop" on "locksmithd.service"
Jan 26 16:58:33 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:33 Result of "stop" on "locksmithd.service": done
Jan 26 16:58:33 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:33 Calling unit command "stop" on "fleet.service"
Jan 26 16:58:33 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:33 Result of "stop" on "fleet.service": done
Jan 26 16:58:33 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:33 Calling unit command "stop" on "fleet.socket"
Jan 26 16:58:33 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:33 Result of "stop" on "fleet.socket": done
Jan 26 16:58:33 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:33 Calling unit command "stop" on "flanneld.service"
Jan 26 16:58:33 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:33 Result of "stop" on "flanneld.service": done
Jan 26 16:58:33 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:33 Calling unit command "start" on "systemd-networkd-wait-online.service"
Jan 26 16:58:33 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:33 Result of "start" on "systemd-networkd-wait-online.service": done
Jan 26 16:58:33 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:33 Calling unit command "start" on "k8s-api-server.service"
Jan 26 16:58:34 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:34 Result of "start" on "k8s-api-server.service": done
Jan 26 16:58:34 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:34 Calling unit command "start" on "k8s-controller-manager.service"
Jan 26 16:58:35 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:35 Result of "start" on "k8s-controller-manager.service": done
Jan 26 16:58:35 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:35 Calling unit command "start" on "k8s-scheduler.service"
Jan 26 16:58:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:37 Result of "start" on "k8s-scheduler.service": done
Jan 26 16:58:37 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:58:37 Calling unit command "start" on "k8s-addons.service"
Jan 26 16:59:50 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:59:50 Result of "start" on "k8s-addons.service": done
Jan 26 16:59:50 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:59:50 Calling unit command "stop" on "locksmithd.service"
Jan 26 16:59:50 ip-10-12-1-88.eu-central-1.compute.internal bash[808]: 2018/01/26 16:59:50 Result of "stop" on "locksmithd.service": done
```
From here you can see that when executing our cloud config, all the files in `path` entries are created first, including the certificates, and then the units are created, started and eventually enabled.

So, on first boot, the certificates are created first and the decrypt units are started later. On second boot, if the decrypt units are enabled, it is possible that they are executed before the transient unit that interprets our master.yaml, and because the certificates are already there since the first boot, the condition you propose would be met, and the decrypt units could work on an outdated version of the certificates. This is the race condition, in this case, the decrypt units created by the transient service would not be started because they have been already executed, and the decrypted files available on master would not match the actual certificates.

Maybe I'm not getting right what you propose, WDYT?

Reply:
thanks for detailed explanation. you're right. i did not get that we need new certificates here.

fine for me then.

Reply:
I guess the thing that makes this fix weird is that this is complicated to understand when looking at the unit itself. It isn't declared within the unit but instead bound to the way how cloud-init works.

First boot:
Unit is introduced by cloud-init and started (which happens after files are written to disk)

Second boot:
Unit is already there and systemd would start the unit in no defined order which can lead to a race. We need to take care that the unit is started after the files have been updated.

Another alternative approach would be to define `After=` in the decrypt unit and make sure it is run after the cloud-config. But I am fine with both.

Reply:
@teemow thx a lot for your comments :) 

> Another alternative approach would be to define After= in the decrypt unit and make sure it is run after the cloud-config. But I am fine with both.

Yep, I tried this first, the problem with this approach is how the instance executes cloud-init, it launches first `oem-cloudinit.service`, which pulls the instance's user data, and then a transient unit with a random name, something like `coreos-cloudinit-476003234.service`, which actually runs our `master.yaml`. We should set the `After` dependency directive in the decrypt units to the transient unit, if not the race can still happen, but we don't know the name before-hand.

Reply:
Yes this pretty much sucks. I wonder if they use a target for the cloudconfig that can be used. But anyhow please continue with your approach. We are going to move to ignition anyway.